src/js/background/background.js

```diff
@@ -99,7 +106,8 @@ async function storeRuntimeData() {
   if (typeof existingPremiumAvailability === "object" && 
       typeof existingIntroPricingEndDate === "object" && 
       existingPremiumAvailability.fetchedAt > checkingRemainingDays &&
-      existingIntroPricingEndDate.fetchedAt > checkingRemainingDays
+      existingIntroPricingEndDate.fetchedAt > checkingRemainingDays &&
```

Comment:
We can probably remove the `existingIntroPricingEndDate` stuff on L101, L107 and L109.

Reply:
Removed!